fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/generate_smoke_tests.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+import subprocess
+
+import yaml
+
+
+def search_for_smoke_tests(tests_name):
+    """Searches for smoke tests within the requested test module."""
+    path = Path("../ui")
+    files = sorted([x for x in path.iterdir() if x.is_file()])
+    locations = []
+    file_name = None
+    test_names = []
+
+    for name in files:
+        if tests_name in name.name:
+            file_name = name
+            break
```

Comment:
Maybe `next()` could be used here?

I think it's technically possible for `file_name` to remain `None` after this loop. Perhaps add a check before opening the file?